app/src/main/java/org/astraea/metrics/jmx/MBeanClient.java

```diff
@@ -0,0 +1,81 @@
+package org.astraea.metrics.jmx;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import javax.management.*;
+import javax.management.remote.JMXConnector;
+import javax.management.remote.JMXConnectorFactory;
+import javax.management.remote.JMXServiceURL;
+
+public class MBeanClient implements AutoCloseable {
```

Comment:
Did you ignore the method of querying MBeans?

Reply:
Thanks for reminding me, I will implement it later.

Reply:
I implement the querying ability at https://github.com/skiptests/astraea/pull/15/commits/43f8d732d71360e3135604d63b4925d28c93bc6e.